app/src/main/java/org/oppia/app/drawer/NavigationDrawerItem.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.drawer
+
+import org.oppia.app.R
+
+/** Enum to store the menu of [NavigationDrawerFragment] and get menu by id. */
```

Comment:
Maybe: Represents different destinations that can be selected by the user from the navigation drawer.

Reply:
Done.